addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -240,6 +409,9 @@
                 'default_mailing_type': 'sms',
                 'mailing_sms': True
         }</field>
+        <field name="view_ids" eval="[
```

Comment:
If necessary, define real ir.actions.act_window.view as it is more maintainable and easier for inheritance / customization.